UPGRADE-3.1.md

```diff
@@ -24,6 +24,40 @@ FrameworkBundle
 
  * As it was never an officially supported feature, the support for absolute
    template paths has been deprecated and will be removed in Symfony 4.0.
+ * The core form types registered as services:
```

Comment:
Breaking the sentence in the middle looks weird, what about

`The following form types registered as services are deprecated since 3.1 and will be removed in 4.0; use their fully-qualified class name instead:`